src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx

```diff
@@ -24,61 +22,79 @@ export const PriceDatabaseBenefits: React.FC = () => {
+        Image={<SectionImage src="" srcSet="" alt="" />} // TODO: replace with image
+      />
 
       <Separator />
 
-      <GridColumns my={4} gridRowGap={[2, 0]}>
-        {isMobile && (
-          <Column span={6}>
-            <Image style={{ maxWidth: "100%" }} src={"images/browse.jpg"} />
-          </Column>
-        )}
-        <Column span={6}>
-          <Text as="h1" variant="xl">
-            1 Database
-          </Text>
-          <Text lineHeight={1.5}>
-            Quickly and easily find Design, Fine and Decorative Art results all
-            in one place.
-          </Text>
-        </Column>
-        {!isMobile && (
-          <Column span={6}>
-            <Image style={{ maxWidth: "100%" }} src={"images/browse.jpg"} />
-          </Column>
-        )}
-      </GridColumns>
+      <Section
+        title="1 Database"
+        text="Quickly and easily find Design, Fine and Decorative Art results all in one place."
+        Image={<SectionImage src="" srcSet="" alt="" />} // TODO: replace with image
+        imagePosition="right"
+      />
 
       <Separator />
 
-      <GridColumns my={4} gridRowGap={[2, 0]}>
-        <Column span={6}>
-          <Image style={{ maxWidth: "100%" }} src={"images/browse.jpg"} />
-        </Column>
-        <Column span={6}>
-          <Text as="h1" variant="xl">
-            Search for Free
-          </Text>
-          <Text lineHeight={1.5}>
-            No search limits, no results limits, no subscriptions, and no
-            obligations.
-          </Text>
-        </Column>
-      </GridColumns>
+      <Section
+        title="Search for Free"
+        text="No search limits, no results limits, no subscriptions, and no obligations."
+        Image={<SectionImage src="" srcSet="" alt="" />} // TODO: replace with image
+      />
     </Flex>
   )
 }
+
+const Section: React.FC<{
+  title: string
+  text: string
+  Image: ReactElement
+  imagePosition?: "left" | "right"
+}> = ({ title, text, Image, imagePosition = "left" }) => {
+  return (
+    <>
+      <Media lessThan="sm">
+        <GridColumns my={4} gridRowGap={[2, 0]}>
+          <Column span={6}>{Image}</Column>
+          <Column span={6}>
+            <Text as="h1" variant="xl">
+              {title}
+            </Text>
+            <Text variant="sm">{text}</Text>
+          </Column>
+        </GridColumns>
+      </Media>
+      <Media greaterThanOrEqual="sm">
+        <GridColumns my={4} gridRowGap={[2, 0]}>
+          {imagePosition === "left" && <Column span={6}>{Image}</Column>}
+          <Column span={6}>
+            <Text as="h1" variant="xl">
+              {title}
+            </Text>
+            <Text variant="sm">{text}</Text>
+          </Column>
+          {imagePosition === "right" && <Column span={6}>{Image}</Column>}
+        </GridColumns>
+      </Media>
+    </>
+  )
+}
+
+const SectionImage: React.FC<{ src: string; srcSet: string; alt: string }> = ({
+  src,
+  srcSet,
+  alt,
+}) => (
```

Comment:
This is a super minor totally non-blocking nit (I need to update the Best Practices doc about this... will do this now) but if possible, we should always avoid implicit returns on react components. At some point someone somewhere will need to return to this component and debug or augment it in some way, requiring that the implicit return is unwrapped and an explicit `return` is added.

Reply:
Very good idea! What do you think about adding this [ESLint rule](https://eslint.org/docs/rules/arrow-body-style)?

```
 "arrow-body-style": ["error", "always"]
 ```
 
 or
 
 ```
 "arrow-body-style": ["error", "as-needed"]
 ```

Reply:
This issue mostly only applies to React components, so unfortunately I wouldn't want this to be too general and ding one-liners, etc. It's just a good habit to be aware of 😄 

(Also, updated docs yesterday [here](https://github.com/artsy/force/blob/master/docs/best_practices.md#avoid-implicit-returns-on-react-components))